tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/upgrade-e2e-test.go

```diff
@@ -50,6 +50,21 @@ var _ = Describe(
 			})
 		})
 
+		AfterEach(func() {
+			By("Deleting IBGUs on target hub cluster", func() {
+				_, err := ibgu.NewIbguBuilder(cnfinittools.TargetHubAPIClient,
+					tsparams.IbguName, tsparams.IbguNamespace).DeleteAndWait(1 * time.Minute)
+
+				Expect(err).ToNot(HaveOccurred(), "Failed to delete prep-upgrade ibgu on target hub cluster")
```

Comment:
nit: it is no longer "prep-upgrade" ibgu when we clean it up in afterEach. Probably just as simple as Failed to clean up ibgu xxx on target hub cluster.